### PR TITLE
Frontend Homework 11 complete

### PR DESCRIPTION
**Pull request by asish918**
## Drift Video
[Drift](https://video.drift.com/v/absNL3Mg7nJBZPRHh1UvRSBshDayVthCTpm2t7WNVrcE)

## Component Diagram
Quotes-Component

## Screenshots

Desktop View
screencapture-localhost-5173-2024-02-21-22_08_22

Screenshot 2024-02-21 at 22 08 46

Mobile View

Screenshot 2024-02-21 at 22 09 14

Screenshot 2024-02-21 at 22 09 19

### CODE REVIEW

**Comment by asish918** [comment]
| Metric | Value |
| ------ | ----- |
| alert_status | <b>:white_check_mark: OK</b> |
| bugs | 1 |
| code_smells | 0 |
| reliability_rating | B (at least 1 Minor Bug)<br/>Note: A being best, E being worst |
| security_rating | A (0 Vulnerabilities)<br/>Note: A being best, E being worst |
| vulnerabilities | 0 |

**Comment by Nitesh2503** [comment]
The irrelevant files must not be pushed

File: frontend/homework/11/src/components/quotes/Quotes.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { QuoteFilter, QuotesApi } from "../../types"
+
+import './Quotes.scss'
+
+interface QuotesProps {
+    quote: QuotesApi;
+    filterList: QuoteFilter[];
+    setFilterList: React.Dispatch<React.SetStateAction<QuoteFilter[]>>;
+}
```
Comment (Nitesh2503) [inline frontend/homework/11/src/components/quotes/Quotes.tsx:9]: Use separate file for this

File: frontend/homework/11/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    gsap.fromTo(container, { y: 100 }, { y: 0, duration: 1 });
+  }, []);
+
+  useEffect(() => {
+    async function apiCall(): Promise<void> {
+      const res = await fetch("https://api.quotable.io/quotes/random?limit=3");
+      const data: QuotesApi[] = await res.json();
+
+      setAllQuotes(data);
+      console.log(data);
```
Comment (Nitesh2503) [inline frontend/homework/11/src/App.tsx:41]: Avoid unnecessary logs

**Pull request closed without merging**